device-registry/src/test/scala/org/genivi/sota/device_registry/GroupsResourceSpec.scala

```diff
@@ -38,30 +38,32 @@ class GroupsResourceSpec extends FunSuite with ResourceSpec {
 
   test("GET all groups lists all groups") {
     //TODO: PRO-1182 turn this back into a property when we can delete groups
-    val groups = Gen.resize(10, arbitrary[Set[GroupInfo]]).sample.get
-    groups.foreach { case GroupInfo(id, groupName, namespace, groupInfo, discardedAttrs) =>
-      createGroupInfoOk(id, groupName, groupInfo)
+    val groups = Gen.listOfN(10, arbitrary[GroupInfo]).sample.get
+    groups.map { group =>
```

Comment:
Any particular reason for the switch to ```map``` from ```foreach```? Since you don't care for the result it seems ```foreach``` is semantically closer to what you want, but this is not a big deal.